src/components/LandingPage.js

```diff
@@ -0,0 +1,17 @@
+import GetRocket from "./GetRocket";
+import { Link, Route, Routes } from "react-router-dom";
+
+// LANDING PAGE Component
+const LandingPage = () => {
+    return (
+        <section>
```

Comment:
Great job Zach! Should we add another <p> tag to provide a very short introduction of the game?

Reply:
Agree 100% @rberman6